builds/misc/templates/mqtt-perf-setup.yaml

```diff
@@ -0,0 +1,131 @@
+# Template to set up test host (pull and start 'mqttd' container)
+# as well as publisher node and subscriber node (mqtt-benchmark tool writen in go).
+
+jobs:
+- job: setup_host
```

Comment:
Just want to confirm to run this test, we need to have 2 vsts agent for publisher and subscriber, and another machine (connecting by iotedge-mqtt-perf-amd64-conn) to run MQTT broker.  Totally 3 devices, right?

Reply:
Right, but actually 4 devices: 2 vsts agents, one test host amd64 (iotedge-mqtt-perf-amd64-conn) and one test host RPi3 (iotedge-mqtt-perf-arm32v7-conn)

Reply:
To run test for Linux amd64, we need to 2 dedicated VMs for publisher and subscriber, and 1 VM for VSTS agent.  To run test for Linux ARM32, we need to have 2 dedicated RPis for publisher and subscriber, and 1 RPi for VSTS agent.  Do I understand correctly?

Reply:
At the moment we run tests on actual hardware, not VMs. To run a test for Linux amd64 we need 3 dedicated machines (PC): publisher (w/ VSTS agetn), subscriber (w/ VSTS agent) and one test machine w/ broker. Total 3 machines.